analysers/analyser_merge_charging_station_FR.py

```diff
@@ -20,33 +21,171 @@
+        # clean the values of power
+        string_power = str(round(float(power), 2))
+        cleaned_power = (self.remove_trailing_zeros(string_power))
+        if float(power) < max_output_kw:
+            return '{0} kW'.format(cleaned_power)
+
+        else:
+            if float(power) > (max_kw * 1000):
+                return ''
+
+        if float(cleaned_power) > detection_watts and float(cleaned_power) < (
+                max_kw * 1000):
+            cleaned_power = self.remove_trailing_zeros(
+                float(cleaned_power) / 1000)
+            if float(cleaned_power) > max_output_kw:
+                return ''
+            else:
+                if float(cleaned_power) > max_kw:
+                    return '{0} kW'.format(
+                        self.remove_trailing_zeros(
+                            float(cleaned_power) / 1000))
+        else:
+            cleaned_power = float(cleaned_power) / 1000
+
+        if float(cleaned_power) > max_kw:
+            return ''
+
+        if cleaned_power != 0:
+            return '{0} kW'.format(cleaned_power)
+        else:
+            return ''
+
     def __init__(self, config, logger=None):
         Analyser_Merge_Point.__init__(self, config, logger)
         doc = dict(
-            detail = T_(
-'''A car charging station may be here but is not mapped. The list of the
-charging stations comes from a database published by Etalab. This database
-brings together information published by the various local authorities and
-networks in France.'''),
-            fix = T_(
-'''See [the
-mapping](https://wiki.openstreetmap.org/wiki/France/data.gouv.fr/Bornes_de_Recharge_pour_V%C3%A9hicules_%C3%89lectriques)
-on the  wiki. Add a node or add tags if already existing.'''),
-            trap = T_(
-'''The initial information corresponds to recharging pools and devices and not to
-stations, so some values are worth checking in the field. For instance, an open data point
-with `capacity=6` can sometimes match to 3 charging station with `capacity=2`'''))
-        self.def_class_missing_official(item = 8410, id = 1, level = 3, tags = ['merge', 'fix:imagery', 'fix:survey', 'fix:picture'],
-            title = T_('Car charging station not integrated'), **doc)
-        self.def_class_possible_merge(item = 8411, id = 3, level = 3, tags = ['merge', 'fix:imagery', 'fix:survey', 'fix:picture'],
-            title = T_('Car charging station, integration suggestion'), **doc)
-        self.def_class_update_official(item = 8412, id = 4, level = 3, tags = ['merge', 'fix:imagery', 'fix:survey', 'fix:picture'],
-            title = T_('Car charging station update'), **doc)
+            detail=T_(
+                '''A car charging station may be here but is not mapped. The list of the
+                charging stations comes from a database published by Etalab. This database
+                brings together information published by the various local authorities and
+                networks in France.'''),
```

Comment:
This is a multiline string, so the whitespace is part of the string. Please don't indent :)
Same for the other multiline strings.

Reply:
changed the indent on multiple definitions done